tfs/src/main/java/hudson/plugins/tfs/TeamEventsEndpoint.java

```diff
@@ -225,6 +227,16 @@ public void doConnect(
         dispatch(request, response, body);
     }
 
+    @RequirePOST
+    public void doRmwebhook(

```

Comment:
is this called for every REST API call with rmwebhook in the resource? is the name indicative of the registration?

Reply:
This is root for all rmwebhook related changes.. we have register/link not sure what generic name to use. Just using RmWebHook looks good to me as its the root.